eta/models/manifest.json

```diff
@@ -20,6 +20,26 @@
             },
             "date_created": "2019-12-17 08:25:51"
         },
+        {
+            "base_name": "faster_rcnn_nas_coco_2018_01_28",
```

Comment:
Please follow the existing naming convention in this file. I.e., call this `faster-rcnn-nas-coco`.

Also, while you're here, please add faster-rcnn-nas-lowproposals-coco`. It's 3x faster, likely with similarly high accuracy.